### ISSUE TITLE
systemd module

### ISSUE THREAD

**Issue by BrianHicks** [asteris-llc/converge#138]
Should be:
- `systemd.enabled`
- `systemd.disabled`
- `systemd.started`
- `systemd.stopped`

These should take care of issuing `systemctl daemon-reload` as necessary.

This is how I'm currently doing this:

``` hcl
task "enable" {
  check = "systemctl is-enabled etcd"
  apply = "systemctl enable etcd"

  depends = ["file.content.unit", "task.etcd", "task.data"]
}

task "start" {
  check = "systemctl status etcd | grep Active | tee /dev/stderr | grep running"
  apply = "systemctl start etcd && sleep 2"

  depends = ["task.enable"]
}
```

And how I'd like to do it:

``` hcl
systemd.enable "etcd" {
  unit = "etcd"
}

systemd.start "etcd" {
  unit    = "etcd"
  timeout = "2s"

  depends = ["systemd.enable.etcd"]
}
```

**Comment by BrianHicks** [comment]
@rebeccaskinner I think you're working on this now, so I've assigned you. Feel free to unassign if I'm wrong. :)

**Comment by arichardet** [comment]
Implemented by #602
